docs/001_develop/03_client-capabilities/008_interaction/combobox.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: 'Combobox'
+sidebar_label: 'Combobox'
+id: client-interaction-combobox
+keywords: [interaction, interactive, combobox, form, submit, data]
+tags:
+- interaction
+- interactive
+- combobox
+- form
+- submit
+- data
+sidebar_position: 8
+---
+
+import ComboboxDemo from '../../../../examples/ui/client-capabilities/interaction/combobox.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Options from '../../../_includes/_options.mdx'
+
+# Combobox
+
+A combo-box is an input with an associated pop-up that enables users to select one value from a collection of possible values.
+
+Use cases:
+* Selecting pre-defined values
+* Sorting
+* Filtering
+
+## Example
+
+<ComboboxDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+  <TabItem value="genesis">
+
+    Declaration
+    ```html
+    <rapid-combobox>
+      <rapid-option value="s">Small</rapid-option>
+      <rapid-option value="m">Medium</rapid-option>
+      <rapid-option value="l">Large</rapid-option>
+    </rapid-combobox>
+    ```
+
+    Usage
+    ```typescript
+    @customElement({
+      name: 'my-element',
+      template: html`
+        <rapid-combobox @change=${x => x.selectChanged()}>
+          <rapid-option value="s">Small</rapid-option>
+          <rapid-option value="m">Medium</rapid-option>
+          <rapid-option value="l">Large</rapid-option>
+        </rapid-combobox>
+      `,
+    })
+    export class MyElement extends GenesisElement {
+      selectChanged() {
+        alert('Select Changed')
+      }
+    }
```

Comment:
Can you tweak this example so it saves the current combobox state as part of the example